chapters/functions.tex

```diff
@@ -2398,6 +2398,11 @@ \subsection{Annotations for External Functions}\label{annotations-for-external-l
   {\lstinline!annotation(SourceDirectory="modelica:/ModelicaLibraryName/Resources/Source")!}\annotationindex{SourceDirectory}, gives the location for source files.
   The preceding one is the default and need not be specified; but another location could be specified by using an URI name for the source directory, see \cref{external-resources}.
   It is not specified how they are built.
+\item
+  The {\lstinline!annotation(License="modelica:/ModelicaLibraryName/Resources/Licenses/MyLicense.txt")!}\annotationindex{License}, gives the license files for the function.
```

Comment:
Would it make sense to give another example of what the URI for an external function could look like, showing that it doesn't have to be the same as the dedicated place for the top-level class?

Reply:
I'm not sure - I understand many projects try to be restrictive for the location of licenses, so I wouldn't want to suggest that.